tests/e2e_operations.py

```diff
@@ -1428,6 +1428,30 @@ def run_initial_tcb_version_checks(const_args):
             assert False, "No TCB_version found in recovery ledger"
 
 
+def wait_for_sealed_secrets(node, min_seqno=0, timeout=10):
+    out, _ = node.remote.get_logs()
+    start = time.time()
+    while True:
```

Comment:
Since this returns on success, it's a bit neater and shorter to have the time check as the while condition.